src/frontend/components/administration/health/HealthTab.vue

```diff
@@ -1,21 +1,13 @@
 <template>
   <v-card>
     <v-toolbar color="secondary" dark flat dense>
-      <v-toolbar-title v-text="$t('administration.health.title')" />
+      <v-toolbar-title> {{ $t('administration.health.title') }} </v-toolbar-title>
       <v-spacer />
       <v-icon>mdi-security</v-icon>
     </v-toolbar>
 
-    <v-row
-      v-if="Object.keys(healths).length === 0"
-      align="center"
-      justify="center"
-      class="ma-2"
-    >
-      <v-col class="text-center" cols="12" sm="4">
-        {{ $t("administration.health.noHealth") }}
-      </v-col>
-    </v-row>
+    <Loader v-if="loading" />
+    <NoData v-else-if="Object.keys(healths).length === 0" :local-key="'administration.health.noHealth'" />
```

Comment:
`healths` may be undefined

```suggestion
    <NoData v-else-if="!healths || Object.keys(healths).length === 0" :local-key="'administration.health.noHealth'" />
```